From 6a57ff26d695aaad096b798879a5dbc5af2cedf5 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 20 Dec 2022 10:46:50 -0800
Subject: [PATCH] configure: Add AC_SYS_LARGEFILE autoconf macro

This will define _FILE_OFFSET_BITS to be 64 if off_t is 64bit
and we do not need to define lfs64 functions

Upstream-Status: Submitted [https://github.com/inotify-tools/inotify-tools/pull/174]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index bddf14d..b89a266 100644
--- a/configure.ac
+++ b/configure.ac
@@ -17,6 +17,9 @@ AC_PROG_CC
 AM_INIT_AUTOMAKE
 LT_INIT
 
+# Add option for largefile support
+AC_SYS_LARGEFILE
+
 AC_PATH_PROG(DOXYGEN, doxygen, NO_DOXYGEN)
 
 AC_ARG_ENABLE(doxygen,
